.prettierrc

```diff
@@ -1,9 +1,11 @@
 {
-  "printWidth": 100,
-  "semi": false,
+  "printWidth": 80,
+  "semi": true,
   "singleQuote": true,
-  "bracketSpacing": false,
-  "jsxBracketSameLine": true,
-  "trailingComma": "none",
+  "bracketSpacing": true,
+  "bracketSameLine": false,
+  "experimentalTernaries": true,
```

Comment:
Do we really want to use experimental stuff?

Reply:
It will be default next version, it's experimental so people can migrate to it beforehand.